tools/jenkins/slaves/Dockerfile.centos

```diff
@@ -38,9 +38,10 @@ RUN scl enable rh-ror50 $HOME/bushslicer/tools/hack_bundle.rb
 #   security concern because of exposing VPN details
 # RUN sed -i -e 's/^\s\+curl\s/  curl -Sk /' /usr/local/bin/run-jnlp-client
 
+# Cleanup and remove any instance of the private repository
 RUN rpm -qa && \
     yum clean all -y && \
-    rm -rf $HOME/bushslicer /var/cache/yum /tmp/*
+    rm -rf /var/cache/yum /tmp/* $HOME/bushslicer/private* $HOME/bushslicer/tierN
```

Comment:
`$HOME` needs to be quoted to make sure there are no spaces in it. Unlikely especially in this context, but better safe than sorry.

```
rm -rf /var/cache/yum /tmp/* "$HOME"/bushslicer/private* "$HOME"/bushslicer/tierN
```

I'm a little paranoid but I believe this one time is justified :innocent: